src/plugins/data/public/search/sync_search_strategy.ts

```diff
@@ -31,24 +31,26 @@ export interface ISyncSearchRequest extends IKibanaSearchRequest {
 
 export const syncSearchStrategyProvider: TSearchStrategyProvider<typeof SYNC_SEARCH_STRATEGY> = (
   context: ISearchContext
-) => {
+): ISearchStrategy<typeof SYNC_SEARCH_STRATEGY> => {
   const search: ISearch<typeof SYNC_SEARCH_STRATEGY> = (
     request: ISyncSearchRequest,
     options: ISearchOptions = {}
   ) => {
+    const loadingCount$ = new BehaviorSubject(0);
+    context.core.http.addLoadingCountSource(loadingCount$);
```

Comment:
You're going to keep adding an observable for every search.
I think we should just add it once and only trigger it for every search.

Reply:
Good call, resolved in 9a7a651988.